### ISSUE TITLE
Attempting to configurable attribute of unconfigurable property. 

### ISSUE THREAD

**Issue by yolofy** [brigand/react-safe-html#8]
Getting this error in the console on iPhone 4s Safari browser when using this component.

`Attempting to configurable attribute of unconfigurable property. `

**Comment by brigand** [comment]
@yolofy please provide a test case, with your code and the html that causes the issue.

**Comment by yolofy** [comment]
Just load the component. The problem is in index.js#26:
```
Object.defineProperty(C, 'name', {
    configurable: true,
    value: `SafeHtml__${tag}`,
  });

  return C;
}
```
Is it necessary to make the name property configurable? Please google the error message, it's really common.

**Comment by ljharb** [comment]
Indeed; in order for it to be safe to define a function’s name property across browsers, you need to first determine if they’re configurable on the platform.

**Comment by brigand** [comment]
Fixed in 0.6.1
